module/VuFind/src/VuFind/Log/Handler/PostHandler.php

```diff
@@ -101,26 +87,33 @@ public function setContentType($type)
     protected function getBody($event)
     {
         return json_encode(
-            ['message' => $this->formatter->format($event) . PHP_EOL]
+            ['message' => $event['message'] . PHP_EOL]
         );
     }
 
     /**
-     * Write a message to the log.
+     * Writes the record down to the log
      *
-     * @param array $event event data
+     * @param LogRecord $record LogRecord
      *
      * @return void
-     * @throws \Laminas\Log\Exception\RuntimeException
      */
-    protected function doWrite(array $event)
+    protected function write(LogRecord $record): void
     {
-        // Apply verbosity filter:
+        $event = [
+            'timestamp' => $record->datetime,
+            'priority' => $record->level->value,
+            'priorityName' => $record->level->getName(),
+            'message' => $record->formatted,
```

Comment:
You could probably eliminate the if statement following this assignment by changing this line:

```suggestion
            'message' => is_array($record->formatted) ? $record->formatted[$this->verbosity] : $record->formatted,
```